ui.apps/src/main/content/jcr_root/apps/hclecomm/components/content/productdetails/clientlibs/js/detailsofproduct.js.dir/.content.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
@Rajamouli-hcl  why u need this. please delete this.

Reply:
deleted js.dir.flle as per your comment